README.md

```diff
@@ -37,6 +40,26 @@ Run all tests, including usage examples: `go test -tags examples ./...`
 Install linters runner: [golangci-lint local installation](https://golangci-lint.run/usage/install/#local-installation)  
 Run golangci-lint: `golangci-lint run ./...`
 
+Anybody who wants to contribute to development is welcome to provide pull requests.
+
+Subset of the code is generated from API specification available at:
+- <https://github.com/kentik/api-schema-public/tree/master/gen/openapiv2/kentik/cloud_export/v202101beta1>
+- <https://github.com/kentik/api-schema-public/tree/master/gen/openapiv2/kentik/synthetics/v202101beta1>
+
+The OpenAPI Generator is used for code generation: <https://openapi-generator.tech/>.
+Generated code is checked-in to the repository, so that the user can _go get_ the library.
+
+Additional development steps for API v6:
+- Access apiv6 directory: cd apiv6
+- Generate the client: `./generate_client.sh`
+- Generate the stub server: `./generate_server.sh`
+- Generate documentation: `./generate_docs.sh`
+
+Note that due to the design of OpenAPI generator, some generated files of the stub server need to be filled manually. They are listed in following files:
```

Comment:
Hmm this is not a "design of the the generator" problem, this is common across such generators that they generate only scaffolding that the programmer needs to fill with actual code, I would rephrase to:
_Note that the generated code scaffolding needs to be complemented with actual logic implementation_